MBBSEmu.Tests/ExportedModules/Majorbbs/hexopt_Tests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MBBSEmu.Module;
+using System.Collections.Generic;
+using System.Text;
+using MBBSEmu.Memory;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class hexopt_Tests : ExportedModuleTestBase
+    {
+        private const int HEXOPT_ORDINAL = 339;
+
+        [Theory]
+        [InlineData("FF", 255, 0, 256)]
```

Comment:
maybe add a test of floor/ceiling functionality to cover those cases?

e.g. `[InlineData("D", ?, 0xE, 0xF]` = casastro?

Reply:
It'd throw an exception if it's outside the range, not just catastro. We could wrap the test in a try/catch and pass in a bool if an exception is expected.